test/accessibility/services/a11y/a11y.ts

```diff
@@ -65,16 +68,28 @@ export class AccessibilityService extends FtrService {
     };
   }
 
-  private assertValidAxeReport(report: AxeReport) {
+  private assertValidAxeReport(report: AxeReport, skipFailures?: boolean) {
     const errorMsgs = [];
 
     for (const result of report.violations) {
       errorMsgs.push(printResult(chalk.red('VIOLATION'), result));
     }
 
-    if (errorMsgs.length) {
+    // Throw a new Error if not skipping failures
+    if (!skipFailures && errorMsgs.length) {
```

Comment:
I used this same logic to update the EUI Cypress functional tests in PR https://github.com/elastic/eui/pull/6173